app/controllers/page_views_controller.rb

```diff
@@ -31,14 +31,7 @@ def create
   end
 
   def update
-    if session_current_user_id
-      page_view = PageView.order(created_at: :desc)
-        .find_or_create_by(article_id: params[:id], user_id: session_current_user_id)
-
-      unless page_view.new_record?
-        page_view.update_column(:time_tracked_in_seconds, page_view.time_tracked_in_seconds + 15)
-      end
-    end
+    Articles::PageViewUpdater.call(article_id: params[:id], user_id: session_current_user_id) if session_current_user_id
```

Comment:
Nitpick (style): Maybe it's just me but this modifier `if` is a bit far to the right of this line. I think I'd prefer a regular `if` here. Or maybe even add an early return in `Articles::PageViewUpdater` so we can call it unconditionally and the controller doesn't need to know anything about page view logic.